### DISCUSSION TITLE
Translation: Missing strings in fr.po

### DISCUSSION THREAD

**Discussion by mathieumd** [sheltermanager/asm3#1354] [General]
It looks like quite a few strings are untranslated, but they are also missing from `fr.po`.

By example, in `options.js` the following strings are absent:
- Boarding
  - Boarding payment type
- Costs
  - The daily cost for every day a shelter animal is in care
  - The cost type used when creating a cost record of the total daily boarding cost for adopted animals
- Data Protection
  - Never anonymize people who adopted an animal
  - Remove animal media this many years after the animal dies or leaves the shelter
  - Remove people with a cancelled reservation who have not had any other contact after this many years
- Defaults
  - Diary Person
  - Log Filter
  - System Log Type
  - Return Reason
  - When I change the flags on an animal or person, make a note of it in the log with this type
- Diary and Messages
  - Auto complete diary notes linked to animals when they are marked deceased
- Display
  - Show animal thumbnails in clinic books
- Events
  - (All)
- etc.

**Comment by bobintetley** [comment]
This is how po files work. When translating them, you update them to add any missing strings from the template file (in this case po/asm.pot)

This can be done using a tool like POEdit under Translation->Update from POT file 
![image](https://github.com/sheltermanager/asm3/assets/18594812/f897eefa-a73e-4b25-99ff-ef80ecab85b2)
